src/Telemetry/Microsoft.DotNet.AzureDevOpsTimelineTests/BuildLogScraperTests.cs

```diff
@@ -0,0 +1,56 @@
+using Microsoft.DotNet.Internal.Testing.Utility;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.AzureDevOpsTimeline.Tests
+{
+    public class BuildLogScraperTests
+    {
+        private Mock<ILogger<BuildLogScraper>> _logger = new Mock<ILogger<BuildLogScraper>>();
```

Comment:
Stateful variables in test classes can lead to some weird interdependencies.  It's better to create each object in the test that uses them.

Also, consider using one of the NUnitLoggers we have, so that when tests fail, we have the logs available in the test output to maybe see what's happening.

Reply:
This whole class was refactored, using DI now and the NUnitLogger